bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -977,26 +967,8 @@ protected void addThingHandlerFactory(ThingHandlerFactory thingHandlerFactory) {
 
     private void registerAndInitializeHandler(final Thing thing, final ThingHandlerFactory thingHandlerFactory) {
         if (thingHandlerFactory != null) {
-            try {
-                SafeMethodCaller.call(new SafeMethodCaller.ActionWithException<Void>() {
-                    @Override
-                    public Void call() throws Exception {
-                        registerHandler(thing, thingHandlerFactory);
-                        initializeHandler(thing);
-                        return null;
-                    }
-                });
-            } catch (TimeoutException e) {
-                logger.warn("Registering a handler for thing '{}' takes more than {}ms.", thing.getUID(),
-                        SafeMethodCaller.DEFAULT_TIMEOUT);
-            } catch (Exception ex) {
-                ThingStatusInfo statusInfo = buildStatusInfo(ThingStatus.UNINITIALIZED,
```

Comment:
Ok, then I missed looking down the code, I only looked up. I somehow expected that this code here was of any use before, but apparently it wasn't...

Just a question which I asked myself (but which is not directly related to this PR): What happens in the (unlikely, but possible) case that handler.initialize() throws an exception AFTER MORE than 5 seconds?